stress-test/build-machines.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+
+
+if [ $# -eq 0 ]
+  then
+    echo "usage: ./build-machines [number_of_machines] /path/to/server/cert/lamassu_op_root_ca.pem" && exit 1
+fi
+
+case $1 in
+    ''|*[!0-9]*) echo "usage: ./build-machines [number_of_machines] /path/to/server/cert/lamassu_op_root_ca.pem" && exit 1;;
+esac
+
+SERVER_CERT=$(perl -pe 's/\n/\\n/' < $2)
+if [ -z "$SERVER_CERT" ]
+  then
+    echo "Lamassu-op-root-ca.pem is empty" && exit 1
+fi
+
+# Remove old folders
+rm -rf ./machines/*
+
+START=1
+END=$1
+for (( c=$START; c<=$END; c++ ))
+do
+  NUMBER=$c
+  mkdir -p ./machines/$NUMBER/
+  cp ../data/client.sample.pem ./machines/$NUMBER/
+  cp ../data/client.sample.key ./machines/$NUMBER/
+
+
+  cat > ./machines/$NUMBER/connection_info.json << EOL
+  {"host":"localhost","ca":"$SERVER_CERT"}
+EOL
+
+  echo 'Generating certs...'
+  node ./init-cert.js $NUMBER
+
+  # Get device_id
+  DEVICE_ID=`openssl x509 -outform der -in ./machines/$NUMBER/client.pem | sha256sum | cut -d " " -f 1`
+
+  # Update db config
+  sudo -u postgres psql -t -d lamassu -c "select data from user_config where type='config' order by id desc limit 1" > config.json
```

Comment:
Can we use a special ``lamassu-stress`` dbname for this? No point polluting the regular dev db.